README.rst

```diff
@@ -941,6 +941,15 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_last`` (``bool``): whether ``null`` should be the last type; defaults to
```

Comment:
Indeed.

What about having two options instead, e.g. `first_types` and `last_types` (there are probably better names) that both accept a list of _type categories_? For now, we would only have the `null` category but we could easily extend that list like (e.g. `['objects', 'scalars', 'null']`) without [changing the options in the future](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2825#discussion_r121333649).